### ISSUE TITLE
API URL supplied in .ztdns.toml doesn't seem to work

### ISSUE THREAD

**Issue by jrprince** [uxbh/ztdns#10]
The default URL in .ztdns.toml gives a 404 error. I searched a bit in the ZeroTier documentation and I think that the API is/can be http://127.0.0.1:9993, however when trying that, I just get a 401 Unauthorized response. Perhaps I'm wrong and there are two separate APIs that I'm conflating, but if that's the case, I'm not sure what to use that won't throw a 404.

This is using go1.10.2 on a Raspberry Pi 3 Model B running Raspbian 9.4 with ZeroTier 1.2.8. I do not seem to be having the same issue as reported in #9

**Comment by jrprince** [comment]
Actually, after a fair bit of fiddling, I believe that the API must have just been down at the time. I worked through determining that while 127.0.0.1:9993 while using the same auth token that zerotier-cli uses works, the API is not the same as what's located at my.zerotier.com/api…when it's actually responding.

Ultimately, it just looks like the service that provides that just wasn't available earlier. Now it's up, and I did get the same error as #9, but switching to that branch got it working. Thanks for a great little tool.
